lib/takeout.rb

```diff
@@ -0,0 +1,43 @@
+require 'twilio-ruby'
+
+ACCOUNT_SID = 'ACd5a9f1864bca50ee0aca797630101375'
+AUTH_TOKEN = '1dc40490bfb8ad122c5dd7a90427780b'
+TWILIO_SENDER = Twilio::REST::Client.new(ACCOUNT_SID, AUTH_TOKEN).messages
+
+class TakeOut
+
+  MENU = {
+  "chicken soup" => 5,
+  "icecream" => 3
+  }
+
+  def initialize(sender = TWILIO_SENDER)
+    @sender = sender
+  end
+
+  def show_menu
+    MENU
+  end
+
+  def process(order)
+    raise "Invalid order" unless valid order
+
+    @sender.create(
```

Comment:
Your Takeout class seems to be doing a lot of work. Could you separate the concerns of the Takeout class from your SMS logic? Perhaps the SMS logic needs to live in its own class?